cmd/application.go

```diff
@@ -65,24 +65,30 @@ func init() {
 
 func runAppDeploy(cmd *cobra.Command, args []string) error {
 	name := args[0]
-	c := config.Get()
-	client := api.NewClient(c.URL, c.APIKey)
+	config, configErr := config.Get()
```

Comment:
Theres like 20 of these. I simply added an `error` to the `config.Get()` function so we can propagate an error when ENVs aren't set instead of panicking.